PathOfBuildingCommunity/PathOfBuilding
# Add support for conditional Quantity/Rarity modifiers & Bisco's

### Description of the problem being solved:
Adds support for IIQ / IIR vs frozen/shocked enemies. (Windripper)
Adds support for Bisco's IIQ / IIR modifiers for normal / magic enemies.

### Steps taken to verify a working solution:
- Add a windripper / biscos to the pob
- Enable in the config enemy is frozen/shocked
- Check the IIQ/IIR in the calcs tab under "Other effects".
- If Bisco's is equiped there should be 2 new rows appearing.

### Link to a build that showcases this PR:
https://pobb.in/jZXMaVk-wD6I

### Before screenshot:
![image](https://github.com/user-attachments/assets/b4086052-e626-41ff-b559-d2bdb9884f29)

### After screenshot:
![image](https://github.com/user-attachments/assets/8c207742-c323-43cf-93f4-ca98c9142916)
![image](https://github.com/user-attachments/assets/a6fbebdf-798e-47a9-94d8-4f08e0b99121)
![image](https://github.com/user-attachments/assets/ee30e2d4-4851-426c-8508-752fbba78f4a)
![image](https://github.com/user-attachments/assets/657e339f-49ed-43bd-a63d-fc2e60b2e286)
